ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -249,6 +258,103 @@ export default {
+      // TODO: Callout spawn location as well (north/south)
+      netRegex: NetRegexes.startsUsing({ source: 'Bozjan Phantom', id: '57BA', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Bozja-Phantom', id: '57BA', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Fantôme Bozjien', id: '57BA', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ボズヤ・ファントム', id: '57BA', capture: false }),
+      delaySeconds: 5,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Unavoidable Knockback',
+          de: 'Unvermeidbarer Rückstoß',
+          fr: 'Poussée inévitable',
+          ko: '넉백 방지 불가',
+        },
+      },
+    },
+    {
+      id: 'DelubrumSav Phantom Vile Wave',
+      netRegex: NetRegexes.startsUsing({ source: 'Bozjan Phantom', id: '57BF', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Bozja-Phantom', id: '57BF', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Fantôme Bozjien', id: '57BF', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ボズヤ・ファントム', id: '57BF', capture: false }),
+      response: Responses.getBehind('alert'),
+    },
+    {
+      id: 'DelubrumSav Phantom Ice Spikes',
+      // Ice Spikes (effectId: '9E0') reflects damage, wait for Dispel
+      // Buff expires about 16 seconds on first cast, ~8 seconds later casts)
+      netRegex: NetRegexes.startsUsing({ source: 'Bozjan Phantom', id: '57BC', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Bozja-Phantom', id: '57BC', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Fantôme Bozjien', id: '57BC', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ボズヤ・ファントム', id: '57BC', capture: false }),
+      delaySeconds: 3,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Stop Attacking, Wait for Dispel',
+        },
+      },
+    },
+    {
+      id: 'DelubrumSav Phantom Excruciation',
+      netRegex: NetRegexes.startsUsing({ source: 'Bozjan Phantom', id: '57BE' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Bozja-Phantom', id: '57BE' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Fantôme Bozjien', id: '57BE' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ボズヤ・ファントム', id: '57BE' }),
+      condition: tankBusterOnParty,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'DelubrumSav Avowed Wrath Of Bozja',
+      netRegex: NetRegexes.startsUsing({ source: 'Trinity Avowed', id: '594E' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Trinität Der Eingeschworenen', id: '594E' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Trinité Féale', id: '594E' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'トリニティ・アヴァウド', id: '594E' }),
+      response: Responses.tankCleave('alert'),
```

Comment:
This one needs more special handling, I think.  Maybe if you are a tank say `triple shared tank cleave` or just `shared tank cleave`, healers can get `tank cleave` and everybody else `avoid tank cleave`?

Reply:
I opted for shared tank cleave to keep it short. Unless we want to also clarify the shared tank cleave in Trinity Seeker to be double, I think shared tank cleave is fine here. One could also interpret shared tank cleave to default as '2 tanks' or 'all tanks', but I don't particularly have a preference as I do not play tank role...